.github/workflows/bump-pr-version.yml

```diff
@@ -0,0 +1,68 @@
+name: Bump PR chart version
+on:
+  pull_request:
```

Comment:
On all events on PRs?

Reply:
Definitely on the three defaults (`[opened, reopened, synchronized]`).

Reply:
Yes. It's too late on `closed`